pkg/scalers/pulsar_scaler.go

```diff
@@ -5,15 +5,16 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"io"
-	"net/http"
-	"strconv"
-	"strings"
-
 	"github.com/go-logr/logr"
+	"golang.org/x/oauth2/clientcredentials"
```

Comment:
please keep imports separated in these 3 groups: 
- internal imports 
- external libraries
- KEDA packages